### PR TITLE
Convert the container module to use i18n messages.

### PR DESCRIPTION
**Pull request by jamezp**
This adds JBoss Logging and JBoss Logging Tools dependencies to the container module. The messages have been converted to use the interface message bundle for internationalization support.

Messages that just wrapped and rethrew other messages I did not convert. In some use cases it shouldn't need to be done, but I didn't look extensively at all use cases where it was done.

### CODE REVIEW

File: container/src/test/java/org/wildfly/swarm/internal/ArtifactManagerTest.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public void testArtifactInvalidGroupArtifactVersion() throws Exception {
             manager.artifact(String.format("%s:%s:%s", ARTIFACT_INVALID_GROUP_ID, ARTIFACT_INVALID_ARTIFACT_ID, ARTIFACT_INVALID_VERSION));
             fail("RuntimeException should have been thrown");
         } catch (RuntimeException e) {
-            assertThat(e).hasMessage("Artifact 'no.such:thingy:1.0.0' not found.");
+            assertThat(e).hasMessage("WFSWARM0013: Artifact 'no.such:thingy:1.0.0' not found.");
```
Comment (jamezp) [inline container/src/test/java/org/wildfly/swarm/internal/ArtifactManagerTest.java:123]: Note tests like this will break once translations are available.

**Comment by jamezp** [comment]
I forgot to add this does not solve the internationalization problem with the `BootstrapLogger`. That will need to be solved in a different way.

**Comment by bobmcwhirter** [comment]
You rock!

**Comment by kenfinnigan** [comment]
ok to test

**Comment by kenfinnigan** [comment]
doh, ignore me, it already tested it

**Comment by kenfinnigan** [comment]
Thanks for this @jamezp!

**Pull request merged**